package.json

```diff
@@ -27,6 +27,7 @@
   ],
   "scripts": {
     "preinstall": "node -e \"process.exit(process.env.npm_execpath.includes('yarn') ? 0 : 1)\"",
+    "hops": "cd packages/template-react; yarn hops",
```

Comment:
Do we want to keep this?

Reply:
I would propose to use `yarn workspace hops-template-react hops`.
This is less system specific.